pkg/syncer/metadatasyncer.go

```diff
@@ -2931,26 +2926,12 @@ func csiPVUpdated(ctx context.Context, newPv *v1.PersistentVolume, oldPv *v1.Per
 		_, isdynamicCSIPV = newPv.Spec.CSI.VolumeAttributes[attribCSIProvisionerID]
 	}
 
-	isTopologyAwareFileVolumeEnabled = metadataSyncer.coCommonInterface.IsFSSEnabled(ctx,
-		common.TopologyAwareFileVolume)
-
 	if oldPv.Status.Phase == v1.VolumePending &&
 		(newPv.Status.Phase == v1.VolumeAvailable || newPv.Status.Phase == v1.VolumeBound) &&
 		!isdynamicCSIPV && newPv.Spec.CSI != nil {
 		// Static PV is Created.
 		var volumeType string
 		if IsFileVolume(oldPv) {
-
-			if isMultiVCenterFssEnabled && len(metadataSyncer.configInfo.Cfg.VirtualCenter) > 1 {
-				// If it is a multi VC setup, then skip this volume as we do not support file share volumes
-				// on a multi VC deployment if TopologyAwareFileVolume FSS is not enabled.
-				if !isTopologyAwareFileVolumeEnabled {
-					log.Infof("PVUpdated: %q is a vSphere volume claim in namespace %q."+
-						"File share volumes are not supported in a multi VC setup."+
-						"Skipping PV update.", newPv.Name, newPv.Namespace)
-					return
-				}
-			}
```

Comment:
This seemed to have removed `isMultiVCenterFssEnabled` fss check as well. This also looks like a multi-VC setup scenario, so it doesn't look like we can remove the whole if block. Can you double-check this? I think the right one would be:

```
			if isMultiVCenterFssEnabled && len(metadataSyncer.configInfo.Cfg.VirtualCenter) > 1 {
				// If it is a multi VC setup, then skip this volume as we do not support file share volumes
				// on a multi VC deployment if TopologyAwareFileVolume FSS is not enabled.
				log.Infof("PVUpdated: %q is a vSphere volume claim in namespace %q."+
						"File share volumes are not supported in a multi VC setup."+
						"Skipping PV update.", newPv.Name, newPv.Namespace)
				return
			}
```

Reply:
got it.